src/main/jhipster/src/main/webapp/app/germplasm-manager/germplasm-search.component.ts

```diff
@@ -656,29 +656,37 @@ export class GermplasmSearchComponent implements OnInit {
             return;
         }
 
-        const confirmModalRef = this.modalService.open(ModalConfirmComponent as Component);
-        confirmModalRef.componentInstance.title = 'Delete Germplasm';
-        confirmModalRef.componentInstance.message = 'Are you sure you want to delete the selected germplasm records from the database? The deletion will be permanent.';
-        confirmModalRef.result.then(() => {
-            this.germplasmService.deleteGermplasm(this.getSelectedItemIds()).subscribe((response) => {
-                if (response.germplasmWithErrors && response.germplasmWithErrors.length) {
-                    this.alertService.warning('germplasm-delete.warning');
-                    this.resetFilters();
-                    // Show the germplasm that were not deleted because of validation
-                    this.request.gids = response.germplasmWithErrors;
-                    ColumnFilterComponent.reloadFilters(this.filters, this.request);
-                } else {
-                    this.alertService.success('germplasm-delete.success');
-                }
-                this.resetTable();
-
-                // If there are deleted germplasm, show the Clear Prefix Key Cache Sequence Dialog
-                if (response.deletedGermplasm && response.deletedGermplasm.length) {
-                    this.openKeySequenceDeletionDialog(response.deletedGermplasm);
-                }
-
+        const confirmDeleteGermplasmModalRef = this.modalService.open(ModalConfirmComponent as Component);
+        confirmDeleteGermplasmModalRef.componentInstance.title = 'Delete Germplasm';
+        confirmDeleteGermplasmModalRef.componentInstance.message = 'Are you sure you want to delete the selected germplasm records from the database? '
+            + 'The deletion will be permanent.';
+        confirmDeleteGermplasmModalRef.result.then(() => {
+            this.germplasmService.getGermplasmPresentInOtherLists(this.getSelectedItemIds(), null).subscribe((germplasmPresentInOtherLists) => {
+                const confirmDeleteGermplasmInOtherListsModalRef = this.modalService.open(ModalConfirmComponent as Component);
+                confirmDeleteGermplasmInOtherListsModalRef.componentInstance.title = 'Delete Germplasm';
+                confirmDeleteGermplasmInOtherListsModalRef.componentInstance.message = 'The following germplasm with GIDs: ' + germplasmPresentInOtherLists.join(', ')
```

Comment:
Remove second warning message. The original warning message can just be expanded as:
>  Are you sure you want to delete the selected germplasm records from the database? The deletion will be permanent and can take a long time for germplasm included in lists - from which they will also be deleted.